guidelines/python-cli-ux-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# Python CLI UX Guidelines
+
+1. The tool’s executable name should be “**conjur**” (regardless to its supported platform), so “conjur-py3-cli-darwin” is not a good experience.
+2. Once the CLI tool is executed **without** **any** **commands**, we should show the help and usage.
+3. Make sure to provide short-form options (e.g. -h) and long-form options (--help)
+4. Provide helpful and **readable** **feedback** (output) at all times. E.g., when a command succeeds and doesn’t have output, it is ended (in new line) with: Command succeeded! Data set to variable: XYZ
```

Comment:
I think `Success! Data written to: my_path/user1` so that we can keep it generic because it may also be appropriate for other cases besides variable

Reply:
I agree. done.